packages/twenty-front/src/App.tsx

```diff
@@ -85,36 +87,41 @@ const ProvidersThatNeedRouterContext = () => {
   const pageTitle = getPageTitleFromPath(pathname);
 
   return (
-    <ApolloProvider>
-      <ClientConfigProviderEffect />
-      <ClientConfigProvider>
-        <UserProviderEffect />
-        <UserProvider>
-          <ApolloMetadataClientProvider>
-            <ObjectMetadataItemsProvider>
-              <PrefetchDataProvider>
-                <AppThemeProvider>
-                  <SnackBarProvider>
-                    <DialogManagerScope dialogManagerScopeId="dialog-manager">
-                      <DialogManager>
-                        <StrictMode>
-                          <PromiseRejectionEffect />
-                          <CommandMenuEffect />
-                          <GotoHotkeysEffect />
-                          <PageTitle title={pageTitle} />
-                          <Outlet />
-                        </StrictMode>
-                      </DialogManager>
-                    </DialogManagerScope>
-                  </SnackBarProvider>
-                </AppThemeProvider>
-              </PrefetchDataProvider>
-              <PageChangeEffect />
-            </ObjectMetadataItemsProvider>
-          </ApolloMetadataClientProvider>
-        </UserProvider>
-      </ClientConfigProvider>
-    </ApolloProvider>
+    <>
+      <ApolloProvider>
+        <ClientConfigProviderEffect />
+        <ClientConfigProvider>
+          <WindowEventEffect />
+          <WindowEventProvider>
```

Comment:
One last thing before we merge in the current state, could we rename this into something more specific so that people understand they probably don't have to deal with this? I'm tempted to call it `ChromeExtensionSidecarEffect` and `ChromeExtensionSidecarProvider` or something like that for now. If eventually it becomes something more generic we can rename it?

Reply:
Okay!